msteams-platform/concepts/build-and-test/teams-developer-portal.md

```diff
@@ -60,6 +60,23 @@ Follow the steps to import and manage your existing app in the Developer Portal.
 > * If you create an app using the Microsoft Teams Toolkit for Visual Studio Code, you can manage your app in the Developer Portal. For more information, see [Build apps with teams toolkit and Visual studio code](~/toolkit/visual-studio-code-overview.md).
 > * You can import an existing app which you created on App Studio to the Developer Portal.
 
+## Changelog for Developer Portal
+
+Changelog for Developer Portal let's you stay engaged with the latest updates in Teams. You can view the updates about features, recent changes in APIs, important bug fixes, and other developer products.
+
+To view the Changelog, sign in to [Developer Portal](https://dev.teams.microsoft.com) and select **Changelog** from the left pane.
+
+:::image type="content" source="../../assets/images/tdp/changelog.png" alt-text="Screenshot shows the Changelog in the Developer Portal.":::
+
+The updates for Changelog in the Developer Portal are categorized based on the following:
+
```

Comment:
Remove other developer products.

Added - This service or feature is net new feature/capability 
Removed - This service/API is removed and may be replaced with a new and updated service. 
Deprecated- This service/API is at the end of its life and deprecated. 
Changed - This service/API is modified from <old> to <new> 
Fixed - This service/API that was previously identified as a high-priority/ breaking change issue has been fixed/mitigated 
Security - Important and critical security patch updates